docker-build-v2/build.sh

```diff
@@ -67,10 +99,10 @@ mkdir -p build-$OS .cache/ccache-$OS
 image=recoil-build-amd64-$OS:latest
 if [[ -z "$(docker images -q $image 2> /dev/null)" ]]; then
   image=ghcr.io/beyond-all-reason/recoil-build-amd64-$OS:latest
-  docker pull $image
+  docker pull $DOCKER_PLATFORM_FLAG $image
```

Comment:
This won't make sense if somebody sets `linux/arm64` as flag value suggests, because arch is at the moment part of image name as visible line above.